example.js

```diff
@@ -0,0 +1,43 @@
+// First let's import the logging module.
+var logging = require('./lib/log');
+
+// Now, let's create a module-level object.  The name
+// parameter is mandatory and should be the name of your
+// app.  The object passed in here is just a bunyan log
+// config object.
+var log = logging("my-example", {module: "main"});
```

Comment:
Picking module as the example property is bad.  The idea is that we have 'name' which is the service name, e.g. 'aws-provisioner'.  Maybe this would be more clear...  What we're saying here is that all loggers and child loggers will log the node_env property.  This is still not a great example since NODE_ENV won't change, so we don't want to log it many times, but that's what goes into the configuration object.  We can also put special things in the object if we wanted to use Bunyan's streams API support.  That stuff is cool, but while we're using Heroku to send raw log lines out to something else, we just want the standard of doing `process.stdout`

var log = logging("aws-provisioner", {node_env: process.env.NODE_ENV});